src/test/java/org/minions/devfund/sergio/movie_databaseTest/ActorTest.java

```diff
@@ -0,0 +1,54 @@
+package org.minions.devfund.sergio.movie_databaseTest;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.minions.devfund.sergio.movie_database.Actor;
+import org.minions.devfund.sergio.movie_database.Movie;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+
+public class ActorTest {
+    private Actor actor_1;
+    private Actor actor_2;
+    private Actor actor_3;
+    private Movie movie_1;
+    private Movie movie_2;
+    private Movie movie_3;
+    private ArrayList<Movie> movieList_1;
+    private ArrayList<Movie> movieList_2;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "movieList_2" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "movieList_2" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)